documentation/selector-file-spec.md

```diff
@@ -44,16 +45,26 @@ The data subset selector file specifies subsets of M-Lab data, either for single
 
 `db_snapshots`: Specifies the snapshot dates (in YYYY-MM-DD format) of the MaxMind databases that are required to resolve IP addresses to providers. This field is optional. If not specified or specified as an empty list, consuming programs should use database snapshots closest in time to the snapshots specified and should suppress warnings to the user about missing snapshots.
 
-`site`: Name of M-Lab site where data was collected.
+`start_times`: List of start times of the window in which to collect test results (in ISO 8601 format). This value must end in `Z` (i.e. only UTC time zone is supported).
 
-`client_provider`: Text name of the client provider. This is the substring that should appear in all AS names when a mapping is performed from this parameter to corresponding ASes. For example, "Verizon" should match AS names "Verizon Online LLC", "MCI Communications Services, Inc. d/b/a Verizon Business", "Verizon Data Services LLC", etc. Supports a limited number of provider metanames that translate to all known AS name queries for that provider:
+`sites` _(optional)_: List of M-Lab site names where data was collected.
+
+`client_providers` _(optional)_: List of client providers based on their name. This is the substring that should appear in all AS names when a mapping is performed from this parameter to corresponding ASes. For example, "Verizon" should match AS names "Verizon Online LLC", "MCI Communications Services, Inc. d/b/a Verizon Business", "Verizon Data Services LLC", etc. Supports a limited number of provider metanames that translate to all known AS name queries for that provider:
 
 * `twc`: Time Warner Cable
 * `centurylink`: CenturyLink
 * `level3`: Level 3 Communications
 * `cablevision`: Cablevision Communications
 
-`start_time`: Start time of the window in which to collect test results (in ISO 8601 format). This value must end in `Z` (i.e. only UTC time zone is supported).
+`client_countries` _(optional)_: The ISO 3166-1 alpha-2 country code associated with the IP address of the measurement client. The geolocation of the client address is recorded within BigQuery at the time of measurement based on MaxMind current to that date.
+
+# Changlog 
+
+## As of version 1.1
+
+* Added `client_countries` property.
+* The properties `metric`, `client_provider` and `site` are now represented by the lists `metrics`, `client_providers` and `sites`. 
+* Made `client_providers`, `client_countries`, and `sites` optional.
```

Comment:
(minor, nitpick) `client_countries` is new so its optional property didn't really change.